webapp/maintenance/README.md

```diff
@@ -37,11 +37,3 @@ $ docker-compose up
 
 And the maintenance mode page or service will be started as well in an own container.
 In the browser you can reach it under `http://localhost:5000/`.
-
-{% endtab %}
-{% tab title="On The Server" %}
-
-How to bring a Kubernetes server into maintenance mode is discriped [here](../../deployment/ocelot-social/maintenance/README.md).
```

Comment:
Maybe we could give a hint to the deploy-rebranding repo? @ulfgebhardt

Reply:
I don't think that is necessary and it would be inconsistent with the other parts of the software where we do not hint specific deployment methods